spec/serializer_spec.rb

```diff
@@ -705,4 +705,28 @@ def serialize_primary(object, options = {})
     xit 'is correctly passed through all serializers' do
     end
   end
+
+  describe 'base_url' do
+    it 'is empty by default' do
+      long_comments = create_list(:long_comment, 1)
+      post = create(:post, long_comments: long_comments)
+      data = MyApp::PostSerializer.serialize(post)
+      expect(data['data']['links']['self']).to eq('/posts/1')
+      expect(data['data']['relationships']['author']['links']).to eq({
+        "self"=>"/posts/1/relationships/author",
```

Comment:
Just a style nitpick, use single quotes instead of double quotes for consistency and put spaces around the hash-rocket, here and below.